internal/resources/kubernetessecret/scope/scope.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package scope
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"golang.org/x/exp/slices"
+
+	secretclustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/kubernetessecret/cluster"
+)
+
+var (
```

Comment:
Why in a new line? if there is only 1 variable defined, let's have it like `var ScopeSchema = &schema.Schema{`

Reply:
updated PTAL